docs/sources/administration/planning-iam-strategy/index.md

```diff
@@ -0,0 +1,213 @@
+- You need to rotate or replace a compromised token.
+
+{{% admonition type="note" %}}
+In Grafana's audit logs it will still show up as the same service account.
+{{% /admonition %}}
+
+Service account access tokens inherit permissions from the service account.
+
+### API keys
+
+> **Note:** If you use Grafana v8.5 or newer, you should use service accounts instead of API keys. API keys will be deprecated in the near future. For more information, refer to [Grafana service accounts]({{< relref ".#service-accounts" >}}).
+
+You can use Grafana API keys to interact with data sources via HTTP APIs.
+
+## How to work with roles?
+
+Grafana utilizes roles to control the access of users and service accounts to specific resources and determine their authorized actions.
+
+You can assign roles through the user interface or APIs, establish them through Terraform, or synchronize them automatically via an external IAM provider.
+
+### What are roles?
+
+Within an organization, Grafana has established three primary roles - organization administrator, editor, and viewer - which dictate the user's level of access and permissions, including the ability to edit data sources or create teams.
+To be a member of any organization, every user must be assigned one specific role.
+
+In addition, Grafana offers a server administrator role that grants access to and enables interaction with resources that affect the entire instance, including organizations, users, and server-wide settings.
+This particular role can only be accessed by users of self-hosted Grafana instances. It is a significant role intended for the administrators of the Grafana instance.
+
+### What are permissions?
+
+Each role consists of a set of permissions that determine the tasks a user can perform in the system.
+For example, the **Admin** role includes permissions that let an administrator create and delete users.
+
+Grafana allows for precise permission settings on both dashboards and folders, giving you the ability to control which users and teams can view, edit, and administer them.
+For example, you might want a certain viewer to be able to edit a dashboard. While that user can see all dashboards, you can grant them access to update only one of them.
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}), you can also grant granular permissions for data sources to control who can query and edit them.
+
+Dashboard, folder, and data source permissions can be set through the UI or APIs or provisioned through Terraform.
+
+### Role-based access control
+
+{{% admonition type="note" %}}
+Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise" >}}) and [Grafana Cloud](/docs/grafana-cloud/).
+{{% /admonition %}}
+
+If you feel that the basic organization and server administrator roles are too limiting, it may be beneficial to employ RBAC.
```

Comment:
```suggestion
If you think that the basic organization and server administrator roles are too limiting, it might be beneficial to employ role-based access control (RBAC).
```